ci/rubocop.yml

```diff
@@ -169,7 +169,7 @@ Style/LineEndConcatenation:
   Description: Use \ instead of + or << to concatenate two string literals at line
     end.
   Enabled: false
-Metrics/LineLength:
+Layout/LineLength:
```

Comment:
Is there any reason we want to set 120 for a line length?

Reply:
By default, it was `80`, which is ridiculously low IMHO

Reply:
@abunashir @ronaldtse let's discuss if we have other opinions on this

Reply:
That's understandable, but it does have some great benefits actually, somehow it helps us to keeps the code quite concise, and it's quite useful when you are using small screen with vertical split(like you would normally do in VIM).

I think most of the well known organisation in ruby community like GitHub, Airbnb, Thougtbot or shopify has been using between (80-100) as a default value, so that's also helps to be somewhat syncs with the community.

I would personally prefer to follow these adopted standard, and if we really need to break it for some specific case then we can use the rubocop's inline ignore statement. But, I would leave that upto you, @skalee and @ronaldtse to decide, if you guys feel this the right settings we want then please go ahead.

Reply:
@abunashir thanks for a detailed post, it make sense

@ronaldtse @skalee let us know about your decision

P.S. Nice post about [this "standard"](https://softwareengineering.stackexchange.com/questions/148677/why-is-80-characters-the-standard-limit-for-code-width/148678#148678)

Reply:
I strongly prefer sticking to 80, but I understand sometimes we need exceptions for certain longer lines for deeply nested blocks and haven't been strictly enforcing this across code I have come across (which I used to). 120 would likely wrap my split screen unless I use reduced size fonts. 

Can we stick to 80 and make exceptions as we go? No one is going to reject a PR due to a 100 char-line when it makes sense.

Reply:
Done

Reply:
I support 80 chars limit too. It's not only aesthetics — if line is longer than 80 chars, then usually it is overcomplicated and should be split or refactored in some way. I understand that in some other languages this may be constraining, for example in Java which has long method names, more verbose syntax and wider indentation, but in Ruby usually it's enough.

Note that this limit can be disabled per code block via Rubocop's directives. Also if this limit really doesn't suit you, then you can override it per project or per file in local Rubocop configuration. And I recommend doing exactly that rather than changing it globally. Here is example of per-file override: https://github.com/riboseinc/attr_masker/blob/1f7ea02be5b5ec57db904956534159be7a1d0721/.rubocop.yml#L16-L20

Thanks for reverting this change.